app/_locales/en/messages.json

```diff
@@ -873,6 +873,9 @@
   "history": {
     "message": "History"
   },
+  "iUnderstand": {
```

Comment:
Rather than have the key mimic the text, we should key off of what they represent instead.

Reply:
I like that!